features/configuration/fail_if_no_examples.feature

```diff
@@ -0,0 +1,46 @@
+Feature: fail if no examples
+
+  Use the `fail_if_no_examples` option to make RSpec exit with exit status 1
+  if there are 0 examples. Using this option, it is recommended to add:
+  a `--require spec_helper` option to `.rspec` file to ensure the `fail_if_no_examples`
+  option is set even if no spec files are loaded.
+
+  This option may be particularly useful when you happen to not run RSpec tests
+  locally, but rely on CI to do this. This prevents from false positive builds,
+  when you expected some RSpec examples to be run, but none were run. Such
+   a situation may be caused by your misconfiguration or regression/major changes
+  in RSpec.
+
+  Background:
+    Given a file named "spec/spec_helper.rb" with:
+      """ruby
+      RSpec.configure { |c| c.fail_if_no_examples = true }
+      """
+    Given a file named ".rspec" with:
+      """ruby
+      --require spec_helper
+      """
+    Given a file named "spec/some.spec.rb" with:
+      """ruby
+      RSpec.describe 'something' do
+        it 'succeeds' do
+          true
+        end
+      end
+      """
+
+  Scenario: Examples file has invalid name, thus there are no examples run
+    When I run `rspec`
+    Then it should fail with "0 examples, 0 failures"
+
+  Scenario: Examples file has valid name and 1 example is run
+    Given a file named "spec/some_spec.rb" with:
+      """ruby
+      RSpec.describe 'something' do
+        it 'succeeds' do
+          true
+        end
+      end
+      """
+    When I run `rspec --pattern spec/**/*_spec.rb`
```

Comment:
The pattern you've named here isn't necessary.  It's the default pattern.  My suggestion about passing pattern was to pass a pattern that matches the file you defined in the background, e.g. `--pattern spec/**/*.spec`.  If you do that, there's no need to define a spec file in this scenario -- it can just be a short `When` and a `Then`.  OTOH, if you want to keep the step here defining a spec file, you can remove `--pattern` and remove the step in the background that defines the spec file as it doesn't really serve a purpose.